### DISCUSSION TITLE
May I ask how to change the maximum recruitment limit for crew members?

### DISCUSSION THREAD

**Discussion by BlueSeaAndDragon** [FakeFishGames/Barotrauma#12648] [Code questions and modding support]
May I ask how to change the maximum recruitment limit for crew members? I tried to search for settings such as LuaForBarotrauma or CsForBarotrauma in the game files, but I couldn't find them. If anyone knows the method, please let me know. Thank

**Comment by Regalis11** [comment]
It's a constant called `MaxCrewSize` in `CrewManager`.

**Reply by BlueSeaAndDragon** [reply]
Thank you for your reply, but I still haven't found it... I searched through all the text in Data in the game files; The text in NPCConversations, NPCPrefabs, and Talents in Content. I don't know which text or files I should search for to find it
